### PR TITLE
T1.0 : Task1 is completed

### PR DESCRIPTION
**Pull request by Pattycloud**

### CODE REVIEW

File: git task 3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+third job
```
Comment (manjunath2008) [inline git task 3.txt]: Can u pls add more content after line 1

**Review by manjunath2008 — COMMENTED** [review]
Pls address my review comment

File: git task 3.txt (hunk 1 above)
Comment (Pattycloud) [reply git task 3.txt]: I think the content is good.

Comment (manjunath2008) [reply git task 3.txt]: Nope... please add one line of code..

Comment (Pattycloud) [reply git task 3.txt]: done added new line of code

**Pull request merged**